docs/adoc/microfrontend-platform-developer-guide/chapters/miscellaneous/platform-startup-progress/platform-startup-progress.adoc

```diff
@@ -0,0 +1,18 @@
+:basedir: ../../..
+
+[[chapter:platform-startup-progress]]
+== Platform Startup Progress
+The SCION Microfrontend Platform is started by the class `MicrofrontendPlatform`. You can subscribe to its startup progress to receive feedback and e.g. display a progress bar in your host app while the platform starts.
+
+The following code snippet demonstrates how to subscribe to the startup progress and receive feedback:
+[source,typescript]
+----
+include::platform-startup-progress.snippets.ts[tags=platform-lifecycle:when-state-started]
+----
+
+The platform monitors the progress of following activities:
```

Comment:
of **the** following activities

Reply:
ohne **the** finde ich eigentlich besser.